openmc/stats/univariate.py

```diff
@@ -308,6 +308,163 @@ def to_xml_element(self, element_name):
         element.set("parameters", '{} {}'.format(self.a, self.b))
         return element
 
+class Normal(Univariate):
+    r"""Normally distributed sampling.
+
+    The Normal Distribution is characterized by two parameters
+    :math:`\mu` and :math:`\sigma` and has density function :math:
+    `p(X) dX = 1/\2*sigma\sqrt{\pi} * e^{(X-\mu/\2*sigma}`
```

Comment:
This needs some fixing:
```
:math:`p(X) dX = 1/(\sqrt{2\pi}\sigma) e^{-(X-\mu)^2/(2\sigma)}`
```

In particular, note that `:math:` has to appear next to the backticks.

Reply:
done

Reply:
You still have some slashes in the wrong place (`\2*sigma\sqrt{\pi}` -> `\sqrt{2\pi}\sigma`), and unmatched parentheses within the exponential (looks like there's a `^2` missing inside the exponential as well)